### DISCUSSION TITLE
Docker Build Error

### DISCUSSION THREAD

**Discussion by Bunker56** [PhlexPlexico/G5V#80] [General]
error  in ./src/utils/api.vue

Module Error (from ./node_modules/eslint-loader/index.js):

/G5V/src/utils/api.vue
   403:11  error    Duplicate key 'GetLimitMatches'                   no-dupe-keys
   403:11  error    Duplicated key 'GetLimitMatches'                  vue/no-dupe-keys
   472:11  error    Duplicate key 'DeleteMyCancelledMatches'          no-dupe-keys
   472:11  error    Duplicated key 'DeleteMyCancelledMatches'         vue/no-dupe-keys
   980:29  warning  Replace `(vetoData)` with `vetoData`              prettier/prettier
   981:51  warning  Replace `(vetoSideChoice)` with `vetoSideChoice`  prettier/prettier
   995:40  warning  Delete `,`                                        prettier/prettier
  1002:52  warning  Delete `,`                                        prettier/prettier
  1253:9   warning  Unexpected console statement                      no-console

✖ 9 problems (4 errors, 5 warnings)
  0 errors and 4 warnings potentially fixable with the `--fix` option.

 @ ./src/main.js 12:0-34 14:10-13
 @ multi ./src/main.js

 ERROR  Build failed with errors.
error Command failed with exit code 1.

It`s the latest version of G5V-master, previous was OK.

**Comment by PhlexPlexico** [comment]
Hah, this looks like there was a merge issue from the latest changes from some API stuff that was brought up to date. Will have to look into this later. In the meantime please use the previous commit [here](https://github.com/PhlexPlexico/G5V/commit/83850ccbbca05e2cb42d2bbff6f228ea56ef91e8) for building.

**Reply by PhlexPlexico** [reply]
Latest commit should now have fixed those errors. Let me know if there's any other problems! :)

**Comment by Bunker56** [comment]
everything is OK. I will try it

**Reply by Bunker56** [reply]
Works. It remains only to finish for yourself. Thnx!.. 👍 :)
